### PR TITLE
chore: CODEOWNERS

### LINKED ISSUE #124: [DOC] Add CODEOWNERS
### Change in the documentation

Having `CODEOWNERS` is a [repository requirement](https://github.com/open-feature/.github/blob/main/CONTRIBUTING.md#repository-requirements).

### PR DESCRIPTION
**Pull request by skyerus**
## This PR

Introduces CODEOWNERS (based on contributions).

### Related Issues

Fixes #124

### CODE REVIEW

**Review by james-milligan — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

**Comment by toddbaert** [comment]
I'd like an approval from @davejohnston before we merge this.

@davejohnston I'd be thrilled to have your continued involvement, but I don't want to assume it. There are some expectations associated with this level of maintainership: https://github.com/open-feature/community/blob/main/CONTRIBUTOR_LADDER.md#maintainer

**Review by davejohnston — APPROVED** [review]
LGTM

**Pull request merged**
